bucket/r-devel.json

```diff
@@ -0,0 +1,84 @@
+{
+    "homepage": "https://www.r-project.org",
+    "description": "A free software environment for statistical computing and graphics.",
+    "version": "devel",
+    "license": "GPL-2.0-only",
+    "url": "https://cran.r-project.org/bin/windows/base/R-devel-win.exe",
+    "hash": "183a69e53b35df320a24915af2181b7ebb9f0a335a33f641fcd40d1ed8af21cb",
+    "innosetup": true,
+    "architecture": {
+        "64bit": {
+            "pre_install": "Rename-Item \"$dir\\bin\\R,2.exe\" 'R.exe'",
+            "bin": [
+                "bin\\x64\\R.exe",
+                "bin\\x64\\Rcmd.exe",
+                "bin\\x64\\Rgui.exe",
+                "bin\\x64\\Rscript.exe",
+                "bin\\x64\\Rterm.exe"
+            ],
```

Comment:
Let's keep that and do as the version bucket does : https://github.com/ScoopInstaller/Versions

It will be only one version at a time but easily switchable using scoop and its shims.
```
scoop install r 
# shims will pointing to r now
scoop install r-devel
# shims will be pointing to r-devel now
scoop reset r
# shims have switched back to r 
scoop reset r-devel
# shims have switched back to r-devel
```